DataProc/CPass0/runCPass0.sh

```diff
@@ -336,6 +336,32 @@ if [ $exitcode -ne 0 ]; then
     exit 40
 fi
 
+echo "* Checking the integrity of the residual trees..."
+if [ -f checkResTree.C ]; then
+    echo "Use checkResTree.C macro passed as input"
+else
+    echo "Use checkResTree.C macro from AliDPG"
+    cp $ALIDPG_ROOT/DataProc/CPass0/checkResTree.C .
+fi
+
+echo ""
+echo "running the following checkResTree.C macro:"
+cat checkResTree.C
+echo ""
+echo executing aliroot -l -b -q -x "checkResTree.C()"
+echo "" >&2
+echo "checkResTree.C" >&2
+timeStart=`date +%s`
+time aliroot -l -b -q -x "checkResTree.C" &>> integrity.log
+exitcode=$?
```

Comment:
The ``integrity.log`` will contain the status, but will the validation check it? I guess this log has to be added to the ``/alice/validation/cpass/0/validation.rc``, but should not one also add phrase to grep to the ``validation.sh``? 
Alternatively, one can changed the output in case of the corruption to 
``printf("The residual tree file is corrupted, the job must be Killed\n");``
then this will be intercepted by the current ``validation.sh``. 
I have locally one corrupted chunk, will test it now.

Reply:
The validation.rc lists the files that have to be there, not the ones that are checked. The validation.sh search for the specified strings in all lig files (but the syswatch and tag.log, see https://alimonitor.cern.ch/users/download.jsp?view=true&path=/alice/validation/validation.sh).

Reply:
Ah, and yes, I would add CORRUPTED to the list of strings to check. If you change it to "Killed" it will already work.